ontotext-graphql-playground-component/src/mappers/fetcher-configuration.mapper.ts

```diff
@@ -0,0 +1,26 @@
+import {ExternalGraphqlPlaygroundConfiguration} from '../models/external-graphql-playground-configuration';
+import {Fetcher} from '../graphiql-models/fetcher';
+
+/**
+ * Utility class for converting an external GraphQL playground configuration into a fetcher configuration compatible with GraphiQL.
+ */
+export class FetcherConfiguration {
+  
+  /**
+   * Converts an external GraphQL playground configuration into a fetcher configuration.
+   *
+   * @param {ExternalGraphqlPlaygroundConfiguration} configuration - The external configuration containing the GraphQL endpoint and optional headers.
```

Comment:
You don't need to put types in jsdoc format for typescript files

Reply:
Removed